docs/authentication/social-connections/notion.mdx

```diff
@@ -31,10 +31,10 @@ You need to set a name, a logo, and associate a Notion workspace with it. Make s
 
 ![Configuring integration](/docs/images/authentication-providers/notion/configuring-integration.jpg)
 
-In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)**. From the list of OAuth vendors, toggle on the Notion option. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **Redirect URIs**, as shown below, after changing the integration type to **Public**. Fill any other information required from Notion and click Submit.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Notion**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Paste the value into the **Redirect URIs**, as shown below, after changing the integration type to **Public**. Fill any other information required from Notion and click Submit.
 
 ![Copying values from the Notion dashboard](/docs/images/authentication-providers/notion/copying-values-from-notion-dashboard.jpg)
 
 Copy the **Client ID** and **Client secret** as shown in the above image. Go back to the Clerk Dashboard and paste them into the respective fields.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Notion is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Notion is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Notion is now configured for your instance.
```